temporal-sdk/src/main/java/io/temporal/internal/statemachines/ExecuteLocalActivityParameters.java

```diff
@@ -49,20 +50,23 @@ public class ExecuteLocalActivityParameters {
   private final @Nonnull Duration localRetryThreshold;
   private final boolean doNotIncludeArgumentsIntoMarker;
   private final @Nullable Duration scheduleToStartTimeout;
+  private final @Nonnull AtomicInteger attemptsDuringThisWFT;
```

Comment:
Looking at this pr and at the namings, I think it will be much cleaner to replace it with just an "attemptCompletion" callback. And implement callback in the WorkflowExecutor code.

Reply:
Yep this looks a bit nicer